main.ino

```diff
@@ -7,6 +7,10 @@ char num[20] = {0};
 String CALLING_NUMBER;
 datetimeInfo t;
 
+unsigned long lastStatusCheckTime = 0;
+unsigned long statusCheckIntervalMs = 300000;
```

Comment:
I would suggest to raise this check to once per hour.
Getting SMS every 5 min can be annoying.